src/Microsoft.VisualStudio.LanguageServices.Razor/Editor/DefaultVisualStudioDocumentTrackerFactory.cs

```diff
@@ -69,7 +77,7 @@ public override VisualStudioDocumentTracker Create(ITextBuffer textBuffer)
             }
 
             var filePath = textDocument.FilePath;
-            var tracker = new DefaultVisualStudioDocumentTracker(filePath, _projectManager, _projectService, _editorSettingsManager, _workspace, textBuffer);
+            var tracker = new DefaultVisualStudioDocumentTracker(filePath, _projectManager, _projectService, _editorSettingsManager, _workspace, textBuffer, _documentManager);
```

Comment:
The document tracker is becoming a god-object. https://en.wikipedia.org/wiki/God_object 

This isn't super working.